student.rb

```diff
@@ -0,0 +1,10 @@
+class Student < Person
+  def initialize(age, classroom, name, parent_permission)
+    super(age, name, parent_permission)
+    @classroom = classroom
+  end
```

Comment:
- [ ] You suppose to extend the parent's (person class) constructor with its default values, Kindly fix that.
[Hint] It will be like that:

```ruby
def initialize(age, classroom, name = "Unknown", parent_permission = true)  
    super(age, name, parent_permission)
    @classroom = classroom
  end
```

Reply:
Changes have been made as per instructions